src/__tests__/pages/About.test.js

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+import "@testing-library/jest-dom";
+import { screen, render } from "@testing-library/react";
+import About from "../../pages/About";
+import { MemoryRouter } from "react-router-dom";
+
+describe("About Page", () => {
+  test("renders without crashing", () => {
+    render(
+      <MemoryRouter>
+        <About />
+      </MemoryRouter>,
+    );
+
+    const heading = screen.getByRole("heading", { name: /Our people/i });
```

Comment:
I like the use of "getByRole" here. Being honest, I didn't even know that was a type of role.

Reply:
Thanks, When I was learning about the query types, it was written that getbyrole is always preferable.